.github/workflows/nodejs.yml

```diff
@@ -11,40 +11,45 @@ on:
       - master
 
 jobs:
-  build:
+  test:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         node-version: [12.x]
     steps:
     - uses: actions/checkout@master
+
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: npm install
+
+    - name: Install npm
       run: |
         npm version
-        npm install -g npm
-        npm version
-        npm install
+        npm ci
```

Comment:
`npm ci` is the preferred way to replicate the same package dependency tree in every environment. However I've had numerous issues with CI erroring when `npm ci` is used. I don't mind trying this again. Hopefully it works fine going forward.

Reply:
I hope so,.. if it pops up errors in the future, it can be easily debugged

Reply:
simplifying and cleaning code is my first priority, so, let's watch the behavior